kotlinpoet/src/main/java/com/squareup/kotlinpoet/CodeWriter.kt

```diff
@@ -448,14 +448,27 @@ internal class CodeWriter constructor(
       return memberName.simpleName
     }
 
-    // We'll have to use the fully-qualified name. Mark the member as importable for a future pass.
-    if (!kdoc) {
+    // We'll have to use the fully-qualified name.
+    // Mark the member as importable for a future pass unless the name clashes with
+    // a method in the current context
```

Comment:
nit: missing period.

[Comments should be sentences](https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html).

Reply:
Noted for next time :)